src/ui/SimpleFilter/SimpleFilter.ts

```diff
@@ -115,7 +117,36 @@ export class SimpleFilter extends Component {
      * <div class='CoveoSimpleFilter' data-field='@myotherfield' data-value-caption='{"txt":"Text files","html":"Web page"}'></div>
      * ```
      */
-    valueCaption: ComponentOptions.buildJsonOption()
+    valueCaption: ComponentOptions.buildJsonOption(),
+    /**
+     * See {@link IGroupByRequest.sortCriteria} for a description of each possible value.
+     *
+     * **Allowed values:**
+     *
+     * - `"score"`
+     * - `"occurrences"`
+     * - `"alphaascending"`
+     * - `"alphadescending"`
+     * - `"chisquare"`
+     *
+     */
+    availableSorts: ComponentOptions.buildListOption<'score' | 'occurrences' | 'alphaascending' | 'alphadescending' | 'chisquare'>({
+      defaultValue: ['score'],
+      values: ['score', 'occurrences', 'alphaascending', 'alphadescending', 'chisquare']
+    }),
+    /**
+     * string
+     * Specifies the criteria to use to sort the facet values.
+     *
+     * See {@link IGroupByRequest.sortCriteria} for the list and description of possible values.
+     *
+     * Default value is the first sort criteria specified in the [`availableSorts`]{@link SimpleFilter.options.availableSorts}
+     * option, or `score` if no sort criteria is specified.
+     */
+    sortCriteria: ComponentOptions.buildStringOption({
+      postProcessing: (value, options: ISimpleFilterOptions) =>
+        value || (options.availableSorts.length > 0 ? options.availableSorts[0] : 'score')
```

Comment:
As explained above, this verification should not rely on the "availableSorts"option.